src/cli/README_NETDATA.md

```diff
@@ -379,6 +379,15 @@ Service entries are listed under `Services` header:
 - Context ID
 - Compress flag (`c` if marked or `-` otherwise).
 
+Commissioning Dataset information is printed under `Commissioning` header:
```

Comment:
All the other headers above say "listed" vs "printed" here.  Can you update for consistency

Commissioning Dataset information is listed under...

Reply:
The other entries are lists (we can have zero or more items) but we always have one commissioning dataset.
That's why I used  "printed" for it instead of "listed".